src/elements/hx-progress/index.spec.js

```diff
@@ -0,0 +1,61 @@
+import { fixture, expect } from '@open-wc/testing';
+
+/**
+ * <hx-progress> component tests
+ *
+ * @type HXProgressElement
+ *
+ */
+describe('<hx-progress> component tests', () => {
+    const template = '<hx-progress>';
+
+    describe('instantiate element', () => {
+        it('should be instantiated with hx-defined attribute', async () => {
+            const component = /** @type {HXProgressElement} */ await fixture(template);
+            const attr = component.hasAttribute('hx-defined');
+
+            expect(attr).to.be.true;
+        });
+
+        it('should not be hidden', async () => {
+            const component = /** @type {HXProgressElement} */ await fixture(template);
+            const prop = component.hidden;
+
+            expect(prop).to.be.false;
+        });
+
+        it(`the rendered light DOM should NOT equal simple template ${template}`, async () => {
+            const component = /** @type {HXProgressElement} */ await fixture(template);
+
+            expect(component).lightDom.to.not.equal(template);
+        });
+    });
+
+    describe('test $onConnect method', () => {
+        
+        it('should have aria-valuemin attribute', async () => {
```

Comment:
Let's also add a test to check the value of the `aria-valuemin` attribute, which should be  `0` using `component.getAttribute()`:

```js
        it('aria-valuemin should have a default value of 0', async () => {
            const defaultValue = 0;
            const component = /** @type {HXProgressElement} */ await fixture(template);
            const attr = component.hasAttribute('aria-valuemin');
            const value = component.getAttribute('aria-valuemin');
  
            expect(attr).to.be.true;
            expect(value).to.equal(defaultValue);
        });
```

Reply:
Changes have been done, please review.